benchmarks/llm/prepare.py

```diff
@@ -154,7 +162,7 @@ def main():
 
     #
     huggingface_format = config.get("safetensors", False)
-    pretrained = not args.no_pretrained
+    pretrained = not config.get("no_pretrained", False)
```

Comment:
I am not a fan of "no_xxx" configs, but nothing else comes to mind

Reply:
Or the arg could be `untrained`